ajv/src/ajv.ts

```diff
@@ -63,12 +63,12 @@ export const ajvResolver: Resolver =
 
     const validate = ajv.compile(
       Object.assign(
-        { $async: resolverOptions && resolverOptions.mode === 'async' },
+        { $async: mode === 'async' },
```

Comment:
Ajv resolver was not strictly following the root docs that states that default mode is `async`

I kept the default value as `sync` in order to not change actual behavior for now

Reply:
I totally agree with you, but previous resolver logic basically set it as `sync` by default:
```
(schema, schemaOptions, resolverOptions = {}) =>
...
{ $async: resolverOptions && resolverOptions.mode === 'async' }
```

Can we consider it as bug and set default as `async`? Some users who inadvertently relied on it might have problems

Reply:
Yes, we can consider it as a bug and set default as `async`